crypto/Makefile.in

```diff
@@ -34,7 +34,7 @@ GENERAL=Makefile README crypto-lib.com install.com
 LIB= $(TOP)/libcrypto.a
 SHARED_LIB= libcrypto$(SHLIB_EXT)
 LIBSRC=	cryptlib.c mem.c mem_clr.c mem_dbg.c cversion.c ex_data.c cpt_err.c \
-	ebcdic.c uid.c o_time.c o_str.c o_dir.c thr_id.c lock.c \
+	ebcdic.c uid.c o_time.c o_str.c o_dir.c \
```

Comment:
Oh, I think you need to updated `LIBOBJ` as well to fix classic builds
https://travis-ci.org/openssl/openssl/jobs/114791057#L5991